content/en/blog/_posts/2022-02-16-updated-dockershim-faq.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: blog
+title: "Updated: Dockershim Removal FAQ"
+date: 2022-02-22
+slug: dockershim-faq
+aliases: [ '/dockershim' ]
+---
+
+**This is an update to the original [Dockershim Deprecation FAQ](/blog/2020/12/02/dockershim-faq/) article,
+published in late 2020.**
+
+This document goes over some frequently asked questions regarding the Dockershim
+deprecation [announced as a part of the Kubernetes v1.20 release](/blog/2020/12/08/kubernetes-1-20-release-announcement/). For more detail
+on the removal of Docker as a container runtime for Kubernetes kubelets, and
+what that means, check out the blog post
+[Don't Panic: Kubernetes and Docker](/blog/2020/12/02/dont-panic-kubernetes-and-docker/).
+
+Also, you can read [check whether Dockershim removal affects you](/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/) to determine how impactful dockershim's removal to you or your organization.
+
+As the Kubernetes 1.24 release has become imminent, we've been working hard to try to make this a smooth transition.
+
+- We've written a blog post detailing our [commitment and next steps](/blog/2022/01/07/kubernetes-is-moving-on-from-dockershim/).
+- We believe there are no major blockers to migration to [other container runtimes](/docs/setup/production-environment/container-runtimes/#container-runtimes).
+- There is also a [Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/) guide available.
+- We've also created a page to cover [Articles on dockershim Removal and on Using CRI-compatible Runtimes](/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes/) that include some of the already mentioned docs as well as those from external sources (including vendor guides).
+
+### Why is dockershim being deprecated?
+
+Maintaining dockershim has become a heavy burden on the Kubernetes maintainers.
+The CRI standard was created to reduce this burden and allow smooth interoperability
+of different container runtimes. Docker itself doesn't currently implement CRI,
+thus the problem.
+
+Dockershim was always intended to be a temporary solution (hence the name: shim).
+You can read more about the community discussion and planning in the
+[Dockershim Removal Kubernetes Enhancement Proposal][drkep].
+
+Additionally, features that were largely incompatible with the dockershim, such
+as cgroups v2 and user namespaces are being implemented in these newer CRI
+runtimes. Removing support for the dockershim will allow further development in
+those areas.
+
+[drkep]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2221-remove-dockershim
+
+### Can I still use Docker in Kubernetes 1.20?
+
+Yes, the only thing changing in 1.20 is a single warning log printed at [kubelet]
+startup if using Docker as the runtime.
+
+[kubelet]: /docs/reference/command-line-tools-reference/kubelet/
```

Comment:
I think it's OK to remove this question and its answer. We could instead update it to be relevant for v1.23?

If we keep it, I would revise “Docker” to “Docker Engine”.